pkg/controller/common/helpers.go

```diff
@@ -563,7 +602,7 @@ func TranspileCoreOSConfigToIgn(files, units []string) (*ign3types.Config, error
 	for _, d := range files {
 		f := new(fcctbase.File)
 		if err := yaml.Unmarshal([]byte(d), f); err != nil {
-			return nil, fmt.Errorf("failed to unmarshal file into struct: %v", err)
+			return nil, fmt.Errorf("failed to unmarshal %q into struct: %v", d, err)
```

Comment:
I know this PR didn't originate the `d` but umm what is d and can we change it to something a little clearer? :sweat_smile: